### ISSUE TITLE
Question on Patch-level anomaly score: why use difference + sigmoid instead of traditional softmax?

### ISSUE THREAD

**Issue by InnoFang** [7HHHHH/VisualAD#4]
Thanks a lot for your great work and for open-sourcing the code!

I have a quick question about how you compute the Patch-level anomaly scores.
In most traditional methods, people directly do `softmax` on the dual-channel features, like:
`$logits_i = Softmax(P_i @ t^T)$ for i ∈ {6, 12, 18, 24}`

But in your paper (see the screenshot I attached), you calculate the anomaly score as `$anomaly_sim - normal_sim$`, which can produce negative logits. 
<img width="790" height="219" alt="Image" src="https://github.com/user-attachments/assets/01fde5aa-5656-43a0-853a-0b727e448eb8" />

When upsampling to the anomaly map and computing `Focal Loss` with the ground-truth mask, this might cause issues.
I noticed you handled it in the code like this:
```python
anomaly_map_sigmoid = torch.sigmoid(anomaly_map)
similarity_map = torch.stack([1 - anomaly_map_sigmoid, anomaly_map_sigmoid], dim=1)
```
I'm really curious — **why did you choose this new approach instead of the usual direct softmax for channel normalization?**

Thanks in advance for any explanation!
Best regards!
